helpers/calendar/ratios_and_utilization.py

```diff
@@ -91,22 +90,16 @@ def retrieve_cancellations_and_checkins_for_room(self,
                                                      app_events_list):
         """ Retrieve cancellations and checkins for a room
         :params
-            - calendar_id, start_date, day_after_end_date,
+            - room_id, start_date, day_after_end_date,
                 checkins,cancellations, output
         """
         all_events = CommonAnalytics.get_all_events_in_a_room(
-                self, calendar_id, start_date, day_after_end_date)
+                self, room_id, start_date, day_after_end_date)
         if all_events:
             for event in all_events:
-                event_details = CommonAnalytics.get_event_details(
-                        self, event, calendar_id)
-                if event.get('attendees'):
-                    calendar_events_list.append(event_details)
-                if event.get('organizer') and event.get(
-                        'organizer').get('email') == calendar_id:
-                    app_events_list.append(event_details)
-
-        room_id = RoomModel.query.filter_by(calendar_id=calendar_id,).first().id  # noqa
+                if event['participants']:
+                    calendar_events_list.append(event)
+        room_id = RoomModel.query.filter_by(id=room_id,).first().id  # noqa
```

Comment:
Good work @nakatuddesuzan . It works on my end as expected. But I suppose this `noqa` here can be removed.

Reply:
This has been removed @TeamoreA